src/api/client/resolvers.ts

```diff
@@ -84,7 +84,7 @@ export const queryNodeStitchingResolvers = (
         const viewsLookup = createLookup<{ id: string; views: number }>(batchedVideoViews || [])
         return videos.map((video: Video) => ({ ...video, views: viewsLookup[video.id]?.views || 0 }))
       } catch (error) {
-        ConsoleLogger.warn('Failed to resolve videos field', { error })
+        SentryLogger.error('Failed to resolve videos field', 'batchedVideoViewsResolver', error)
         return null
```

Comment:
If we fail to resolve views but managed to resolve videos, we should return them